src/lib/components/ui/Button/Button.jsx

```diff
@@ -69,14 +72,14 @@ const Button = (props) => {
       type={props.type}
       disabled={props.disabled || props.loading}
     >
-      {(props.icon || props.loading) && (
+      {props.icon && (
         <span
           className={`
             ${styles.icon}
             ${props.loading ? styles.iconLoading : ''}
           `.trim()}
         >
-          <Icon icon={props.loading ? 'sync' : props.icon} size={props.size} />
```

Comment:
This is going to break all our projects. When `loading` is set to `true`, we need to have button with loading icon in it. After your change, `loading` is not used.

Reply:
I think we need another `prop` for `loadingIcon`. it is still a breaking change though…

Reply:
>This is going to break all our projects

You're right. I marked the PR with the _BC_ label.